### PR TITLE
fix chc history

### PR DESCRIPTION
**Pull request by scottty881**
deleting 1927 coverage_history_data_value entries
and 1019 subscriber_data_value entries

### CODE REVIEW

File: dmls_to_run/fix_chc_history.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  returning *;
+  
+  
+DELETE
+FROM
+coverage_history_data_values
+WHERE user_id IN (
+  SELECT users.id 
+  FROM users 
+  INNER JOIN subscriber_data ON subscriber_data.uuid = users.subscriber_uuid
```
Comment (yjhuoh) [inline dmls_to_run/fix_chc_history.sql]: this join won't work if you run the previous query first

Comment (scottty881) [reply dmls_to_run/fix_chc_history.sql]: how about that?

Comment (yjhuoh) [reply dmls_to_run/fix_chc_history.sql]: probably works? it occurred to me while the _values style of data storage is nice for some things, it's really not set up for things like this deletion, especially when we've daisy chained two of them together

Comment (scottty881) [reply dmls_to_run/fix_chc_history.sql]: what do you mean?

Comment (yjhuoh) [reply dmls_to_run/fix_chc_history.sql]: ideally there'd be a way of writing this where order doesn't matter.

put another way, if we ran the query in the original order, we'd have been super screwed.

Comment (scottty881) [reply dmls_to_run/fix_chc_history.sql]: what if we said, ideally we dont drop all of the data

**Pull request merged**